translations/en/front-end-engineer/README.md

```diff
@@ -1 +1,24 @@
-# Front End Engineer path
\ No newline at end of file
+# Career levels the a Front End Engineer
+
+## Junior Front End Engineer 
+
+Now what you know whon your responsibility like Junior Developer, time has come to focus us technical requirements.
```

Comment:
This line is bad translated too.
The correct one should be: "Now, that you know which are your responsibilities as Junior Developer", it's time to focus on..."